src/test/java/com/amihaiemil/zold/PayloadOfTestCase.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2019, Mihai Emil Andronache
+ * All rights reserved.
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 1)Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ * 2)Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * 3)Neither the name of zold-java-client nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.amihaiemil.zold;
+
+import com.amihaiemil.zold.mock.Response;
+import org.apache.hc.core5.http.ContentType;
+import org.apache.hc.core5.http.HttpStatus;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import javax.json.Json;
+
+/**
+ * Unit tests for {@link PayloadOf}.
+ */
+public class PayloadOfTestCase {
+
+    /**
+     * Checks whether the {@link PayloadOf} is instantiated.
+     */
+    @Test
+    public void instantiates() {
```

Comment:
@g4s8 , as it was mentioned, this is the scope of another pull request #53. Regarding your expectation on this, how about the fact that the test is checking that the instantiation is successful? Thus, if the tested constructor fails, the test does not pass. That was why there was no particular test created for the case when the exception is thrown.

Reply:
@denis-konovalyenko so if you test the logic inside constructor, you can just remove assertion, right? I think it will be more clear, because assertion that new object is instance of class is a bit confusing.

Reply:
@g4s8 , I agree with you. This has been corrected within the recent update.

Reply:
@g4s8 , I agree that it would be better to remove the assertion, however, this is related to the pull request #53.
I am really sorry for such complexity. I contributed with the pull request #53 and was hoping that it would be merged within a reasonable time. Moreover, these two pull requests are occupying 2 out of my 3 job slots available and blocking me in moving on with other projects.